src/app/modules/security/signin/signin.component.html

```diff
@@ -0,0 +1,83 @@
+                            </div>
+
+                            <div class="card-panel indigo lighten-5">
+                                <div class="input-field">
+                                    <input formControlName="document_number" id="dni" type="text" class="validate">
+                                    <label for="dni">Número de documento</label>
+                                </div>
+                                <div class="show-errors"
+                                    *ngIf="GetFormGroup['document_number'].errors && (GetFormGroup['document_number'].dirty || GetFormGroup['document_number'].touched)">
+                                    <strong>Este campo es obligatorio y debe contener solo números</strong>
+                                </div>
+                            </div>
+
+                            <div class="card-panel indigo lighten-5">
+                                <div class="input-field">
+                                    <input formControlName="password" id="password" type="password" class="validate">
+                                    <label for="password">Contraseña</label>
+                                    <span toggle="#password" class="material-icons toggle-password"
+                                        style="cursor: pointer; position: absolute; right: 10px; top: 10px;"
+                                        (click)="togglePasswordVisibility()">visibility_off</span>
+                                </div>
+                                <div class="show-errors"
+                                    *ngIf="GetFormGroup['password'].errors && (GetFormGroup['password'].dirty || GetFormGroup['password'].touched)">
+                                    <strong>Este campo es obligatorio</strong>
+                                </div>
+                            </div>
+
+                            <div class="end_form">
+                                <div class="row">
+                                    <div class="col s12">
+                                        <a href="#" class="grey-text text-darken-1">¿Haz olvidado tu contraseña?</a>
+                                    </div>
+                                </div>
+
+                                <div class="row">
+                                    <button (click)="SignIn()" class="btn waves-effect waves-light blue darken-3"
+                                        type="submit" name="action">
+                                        Iniciar sesión
+                                    </button>
+                                </div>
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </form>
+
+        <div class="center-align end">
+            <a href="#" class="black-text">¿Aún no tienes una cuenta?</a>
+            <a href="#"><strong>Crear una cuenta</strong></a>
+        </div>
```

Comment:
This should not be displayed, as a user cannot register himself, he should be registered by medical personnel.